### PR TITLE
Niemeyer2020giraffe saikm200022

### PR DESCRIPTION
**Pull request by saikm200022**

### CODE REVIEW

**Comment by saikm200022** [comment]
@ywen666

File: _summaries/niemeyer2020giraffe_2.md [hunk 1]
```diff
@@ -7,11 +7,34 @@ author: saikm200022
...
-* What interesting variants are explored?
+# **Summary - GIRAFFE: Representing Scenes as Compositional Generative Neural Feature Fields, Niemeyer, Geiger; 2020**
+
+## What is the core idea?
+
+TODO
+
+## How is it realized (technically)?
+
+The paper represents objects in an image as neural feature fields. To allow for disentanglement, they decided to have each object have its own feature field. This is different from previous approaches such as NeRF, because this paper's approach does not represent the entire scene with one model. A transformation from object space to scene space is performed using the below procedure:
```
Comment (ywen666) [inline _summaries/niemeyer2020giraffe_2.md:20]: Maybe describe what neural feature fields in one sentence?

File: _summaries/niemeyer2020giraffe_2.md [hunk 2]
```diff
@@ -7,11 +7,34 @@ author: saikm200022
...
+
+![ALT TEXT](niemeyer2020giraffe_2_1a.PNG)
+
+Next, they rely on the additive nature of the object neural feature fields to create a feature image representation of the scene. This composition operator adds the outputs of the individual feature fields (density values) given a point and its viewing direction denoted x and d respectively:
```
Comment (ywen666) [inline _summaries/niemeyer2020giraffe_2.md:24]: Maybe talk about the motivation to have the composition operator?

File: _summaries/niemeyer2020giraffe_2.md [hunk 3]
```diff
@@ -11,7 +11,9 @@ score: 8
 
 ## What is the core idea?
 
-TODO
+Generating 2D images that are realistic and at high resolutions has been made possible by GANs (Generative Adversarial Networks). However, it is also important to be able to generate these images in a manner that is controllable. The paper suggests an approach, GIRAFFE, that uses a 3D composition representation of scenes that leads to better disentanglement and controllable image generation. This compositional scene approach allows GIRAFFE to be able to learn from raw collections of images without any supplementary supervision.
```
Comment (ywen666) [inline _summaries/niemeyer2020giraffe_2.md:14]: Maybe mention why other methods are not controllable?

**Review by ywen666 — COMMENTED** [review]
Overall a very good review. I proposed a few changes which authors are free to ignore.

**Review by ywen666 — APPROVED** [review]

**Pull request merged**